content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/troubleshooting-the-codeql-workflow.md

```diff
@@ -140,11 +140,11 @@ For more information, see "[Configuring the {% data variables.product.prodname_c
 
 ### Inspect the copy of the source files in the {% data variables.product.prodname_codeql %} database
 You may be able to understand why some source files haven't been analyzed by inspecting the copy of the source code included with the {% data variables.product.prodname_codeql %} database. To obtain the database from your Actions workflow, add an `upload-artifact` action after the analysis step in your code scanning workflow:
-```
+```yaml
 - uses: actions/upload-artifact@v2
   with:
-    name: codeql-database
-    path: ../codeql-database
+    name: {% raw %}codeql-database-${{ matrix.language }}{% endraw %}
+    path: {% raw %}${{ runner.temp }}/codeql_databases{% endraw %}
```

Comment:
I don't get these `{% raw %}` and `{% endraw %}`.  
They don't seem to be interpreted by the GitHub markdown renderer, so they appear in the rendered markdown (and I don't think that was your intent). 

The `${{ matrix.language }}` is rendered correctly regardless. 

```suggestion
    name: codeql-database-${{ matrix.language }}
    path: ${{ runner.temp }}/codeql_databases
```